distribution/src/bin/elasticsearch.bat

```diff
@@ -68,6 +73,26 @@ IF "%checkpassword%"=="Y" (
   )
 )
 
+IF "%attemptautoconfig%"=="Y" (
+    CALL "%~dp0elasticsearch-env.bat" || exit /b 1
+    CALL "%~dp0x-pack-env"
+    CALL "%~dp0x-pack-security-env"
+    ECHO.!KEYSTORE_PASSWORD!| %JAVA% %ES_JAVA_OPTS% ^
+      -Des.path.home="%ES_HOME%" ^
+      -Des.path.conf="%ES_PATH_CONF%" ^
+      -Des.distribution.flavor="%ES_DISTRIBUTION_FLAVOR%" ^
+      -Des.distribution.type="%ES_DISTRIBUTION_TYPE%" ^
+      -cp "!ES_CLASSPATH!;!ES_HOME!/lib/tools/security-cli/*" "org.elasticsearch.xpack.security.cli.ConfigInitialNode" !newparams!
```

Comment:
I discussed the classpath pollution with Albert, and we came up with an alternative. We should remove the call statements above for x-pack-env and x-pack-security-env, and explicitly add those module dirs to the classpath here, so that the ES_CLASSPATH env var is left the same. I also don't think the elasticsearch-env.bat call is necessary, since we would have already done it earlier.

Reply:
Note that this will be very fragile, but no more fragile than the rest of our shell scripts are.

Reply:
Thanks Ryan, that sounds like a plan, I will make the necessary changes here

Reply:
I pushed 1a3220d and verified this works fine on a windows machine